docker-entrypoint

```diff
@@ -64,7 +66,13 @@ term_handler() {
 # shellcheck disable=2039
 trap 'kill $$; term_handler' SIGTERM
 
-if [ "$1" = "autoheal" ] && [ -e "$DOCKER_SOCK" ];then
+if [ "$1" = "autoheal" ]
++then
+  if [ -n "$UNIX_SOCK" ] && ! [ -S "$DOCKER_SOCK" ]
```

Comment:
How is that a valid test for a "connection problem"?

Reply:
You're right. "unix socket is currently not available" is a detailed error message.
-e \"$DOCKER_SOCK\" expression on line 67 raise issue #48 if you use tcp connection since a file "tcp://..." will never exist.
To catch this case i added expression -n "$UNIX_SOCK" because $UNIX_SOCK is empty on tcp connection.

Reply:
Is is possible you meant `test -z` instead of `test -n`?

Reply:
If condition should be true and display message if:
1. $UNIX_SOCK contains a string so we want use unix socket not tcp
and
2. $DOCKER_SOCK is not a socket